src/modules/Tickets/dto/TicketRequest.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable prettier/prettier */
+import { TicketStatus } from "../models/Definitions";
+import { IsEnum, IsNotEmpty, IsOptional, IsString } from "class-validator";
+import { Expose } from "class-transformer";
+import { ITicket } from "@app/data/abstraction/entities/ITicket";
+import { NotAllow } from "@app/decorator/notAllow";
+
+export class CreateTicketStatusRequest implements ITicket {
```

Comment:
Using `@Expose` helps you to preserve the property naming convention as `camelCase` in the code but remain `snake_case` when receiving the request body, please consider to use `camelCase` for property names.